.github/workflows/docker.yml

```diff
@@ -0,0 +1,98 @@
+name: Docker
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  release:
+    types: [published]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  packages: write
+
+jobs:
+  build_and_push:
+    name: Build and Push
+    runs-on: ubuntu-latest
+    strategy:
+      # Prevent a failure in one image from stopping the other builds
+      fail-fast: false
+      matrix:
+        include:
+          - context: "."
+            file: "Dockerfile"
+            image: "librum-server"
+            # ARM not working. Needs further research.
+            # platforms: "linux/arm64,linux/amd64"
+            platforms: "linux/amd64"
```

Comment:
Why is ARM not working? 
Can you be more specific? 
I seam to be able to build an ARM image just fine and 'run it'... just get 404 if i try to open it, so... does require more investigation but not sure you are getting the same issue.

Reply:
Running the workflow with ARM causes an error that I have not been able to resolve. I'm not that well versed in Github Actions. Maybe we can resolve it together. [Here is the error.](https://github.com/tenekev/Librum-Server/actions/runs/7252617194/job/19757468339)

Reply:
Looks to me like it could not find arm  binary probably because there is none. They use x64 binary with a flag...
https://github.com/dotnet/winforms/wiki/Building-for-ARM64#building-on-an-arm64-machine
Here are a possible solution:
https://github.com/NuGet/Home/issues/12227
There are a lot of issues on github that match 'arm64 dotnet dockerfile' and I bet somewhere in there were is a working sample that should integrate with your code.

Reply:
I think I resolved the issue. [Here is a writeup.](https://medium.com/medialesson/building-arm64-docker-images-with-net-8-a-quick-solution-5491a94eb507)

Reply:
Just wanted to add more credibility to @tenekev's write up... it fixed it for me (in 4 separate repositories). Here's one example => https://github.com/LanceMcCarthy/akeyless-web-target/blob/ada27182e26e8eae01c770a10cf21892efcc89f2/src/SecretsMocker/SecretsMocker/Dockerfile#L1